lib/cog/command/command_resolver.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Cog.Command.CommandResolver do
+
+  alias Cog.Repo
+  alias Cog.Queries.Command
+  alias Cog.Queries.Alias
+  alias Piper.Command.SemanticError
+
+  def find_bundle(name) when is_binary(name) do
+    if String.contains?(name, ":") do
+      :identity
+    else
+      case Repo.all(Command.bundle_for(name)) do
```

Comment:
What's the best path to accomplish that? The only thing I can think of is querying for aliases on every command invocation. Is there a more efficient way?